src/index.html

```diff
@@ -5,10 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="header__nav">
+        <a href="#home" class="logo"><img src="../src/images/logo.png" alt="MOYO"></a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item nav__item--gap">
```

Comment:
Remove this classes everywhere
```suggestion
            <li class="nav__item">
```